docs/email-someone-if-shipping-address-fsa-is-out-of-delivery-area/README.md

```diff
@@ -0,0 +1,46 @@
+# Email Someone if Shipping Address FSA is out of delivery area
+
+This checks an order and sends an email to someone if the shipping address FSA isn't included in the FSA list.
+
+* View in the task library: [usemechanic.com/task/email-someone-if-shipping-address-fsa-is-out-of-delivery-area](https://usemechanic.com/task/email-someone-if-shipping-address-fsa-is-out-of-delivery-area)
+* Task JSON, for direct import: [task.json](../../tasks/email-someone-if-shipping-address-fsa-is-out-of-delivery-area.json)
+* Preview task code: [script.liquid](./script.liquid)
+
+## Default options
+
+```json
+{
+  "allowed_fsa__required": "",
+  "email_recipients__required": "",
+  "email_subject__required": "Order {{ order.name }} may be out of delivery zone",
+  "email_body__required_multiline": "Hi team,\n\nPlease see details for order {{ order.name }}:\n\nhttps://{{ shop.myshopify_domain }}/admin/orders/{{ order.id }}\n\nIt looks like this might be out of our delivery zone!\n\nThanks,\n- Mechanic, for {{ shop.name }}"
+}
+```
+
+[Learn about task options in Mechanic](https://docs.usemechanic.com/article/471-task-options)
+
+## Subscriptions
+
+```liquid
+shopify/orders/create
+```
+
+[Learn about event subscriptions in Mechanic](https://docs.usemechanic.com/article/408-subscriptions)
+
+## Documentation
+
+This checks an order and sends an email to someone if the shipping address FSA isn't included in the FSA list.
```

Comment:
Good place to define FSA for folks who aren't Canadian. ;) Here's what I suggest, for this first line:

```
Useful for Canadian stores with limited shipping zones, this task inspects [the FSA (the first three characters of the postal code)](https://en.wikipedia.org/wiki/Postal_codes_in_Canada#Forward_sortation_areas) for each order's shipping address. If an FSA is found that's not on a preconfigured list, this task sends an email to the recipients of your choice.
```